components/ChatPane.vue

```diff
@@ -307,21 +307,8 @@ export default {
 
       return ret
     },
-
-    warnuser() {
-      let ret = false
-      if (this.otheruser) {
-        const user = this.$store.getters['user/get'](this.otheruser.id)
-        if (user && user.info) {
-          const info = user.info
-          ret =
-            info.reneged &&
-            info.reneged > 1 &&
-            (info.reneged * 100) / (info.reneged + info.collected) > 25
-        }
-      }
-
-      return ret
+    userHasReneged() {
+      return this.$store.getters['user/userHasReneged'](this.otheruser.id)
```

Comment:
This is missing the `if (this.otheruser)` check that the previous code had, so potentially `this.otheruser.id` would throw an error.

Reply:
Actually that check would probably be better in the getter maybe.

Reply:
Ah no.  You mean for id...

Reply:
Sorted...